src/fsharp/vs/Symbols.fs

```diff
@@ -1688,6 +1688,32 @@ and FSharpType(cenv, typ:TType) =
        protect <| fun () -> 
         "type " + NicePrint.stringOfTy (DisplayEnv.Empty(cenv.g)) typ 
 
+    member x.Symbol =
```

Comment:
Yep, I can do that on Monday, to be fair I couldn't see an easy way to deal with all the TTypes from an FSharpType.